hive-bigquery-connector-common/src/main/java/com/google/cloud/hive/bigquery/connector/utils/hive/HiveUtils.java

```diff
@@ -149,4 +151,27 @@ public int hashCode() {
       return Objects.hash(getId(), getTaskID().getId(), getJobID());
     }
   }
+
+  /**
+   * Instantiates a new Hive metastore client. The `HiveMetaStoreClient` constructor has different
+   * signatures in different versions of Hive. So we try a few different ways using reflection.
+   */
+  public static HiveMetaStoreClient createMetastoreClient(Configuration conf) {
```

Comment:
can use Hive.get()?
creating a hivemetastoreclient in a hive connector sounds wrong

Reply:
Good point. I changed it to use `Hive.get()`